tests/unit/labels/BinaryLabels_unittest.cc

```diff
@@ -32,3 +34,35 @@ TEST(BinaryLabels,scores_to_probabilities)
 
 	SG_UNREF(labels);
 }
+
+TEST(BinaryLabels, serialization)
+{
+	CBinaryLabels* labels=new CBinaryLabels(10);
+	labels->set_values(SGVector<float64_t>(labels->get_num_labels()));
+
+	for (index_t i=0; i<labels->get_num_labels(); ++i)
```

Comment:
could you instead just make sure that you create an `v = SGVector<float64_t>(labels->get_num_labels())` and set it random... and basically just do:
```
labels->set_values(v);
labels->set_labels(v);
```